releases/release-4.0.15.md

```diff
@@ -0,0 +1,125 @@
+---
+title: TiDB 4.0.15 Release Notes
+category: Releases
+---
+
+# TiDB 4.0.15 Release Notes
+
+Release Date: September 27, 2021
+
+TiDB version: 4.0.15
+
+## Improvements
+
++ TiDB
+
+    - Trigger auto-analyze based on histogram row count [#26706](https://github.com/pingcap/tidb/pull/26706)
+
++ TiKV
+
+    - separate read write ready to reduce read latency [#10619](https://github.com/tikv/tikv/pull/10619)
+
+## Bug Fixes
+
++ TiDB
+
+    - Fix wrong charset and collation for case when function [#26671](https://github.com/pingcap/tidb/pull/26671)
+    - Fix the issue that greatest(datetime) union null returns empty string [#26564](https://github.com/pingcap/tidb/pull/26564)
+    - Fix the issue that sometimes fails to send requests if there are tombstone stores. [#25849](https://github.com/pingcap/tidb/pull/25849)
+
++ TiFlash
+
+    - Fix the potential issue of data inconsistency after crashes when deployed on multi-disks [#2774](https://github.com/pingcap/tics/pull/2774)
```

Comment:
Upgrade guide should be attached here. @JaySon-Huang

Reply:
Do you mean we should make the content in this [doc](https://docs.google.com/document/d/14nhzPiq_fW65cxo9aiYe2mmTgHvkXLt38rMXje2uZiE/edit?n=tics_2770_TiFlash#) published in the website https://docs.pingcap.com/tidb/stable/use-tiflash#use-tiflash? And put a link to that here?